### ISSUE TITLE
Spike Alignment for Tooltip

### ISSUE THREAD

**Issue by iozbeyli** [siemens/ix#1508]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### Suggestion / feature request

Currently the IxTooltip only allows placement of the tooltip around the content. However, in the design file there are more options available for the placement of the spike of the tooltip. It is called spike alignment and allows to place the spike in the corners of the tooltip instead of just to the center. It has "center", "near" and "far" as options. Is this feature going to be available in the library too?

**Comment by danielleroux** [comment]
@iozbeyli can you show me some detailed example? The arrow should always point to the trigger of the tooltip.

**Comment by iozbeyli** [comment]
Yes, currently the tooltip is placing itself according to available space by putting the arrow in the center and that is pointing to the trigger.

![Image](https://github.com/user-attachments/assets/ca618e57-256d-4faf-b685-24923e8d6579)

But in the design file it is possible to create tooltips as in the image above which the arrow is pointing to the trigger but it is not in the center but at the corner. So in other words it is placing the tooltip at bottom left corner instead of bottom or left.

**Comment by nuke-ellington** [comment]
The Figma library clearly marks these options (clear and far) as deprecated. Therefor they will not be implemented.
